SmartDeviceLink-iOS/SmartDeviceLink/SDLProxy.m

```diff
@@ -295,6 +322,12 @@ - (void)handleRegisterAppInterfaceResponse:(SDLRPCResponse *)response {
     //Print Proxy Version To Console
     NSString *logMessage = [NSString stringWithFormat:@"Framework Version: %@", self.proxyVersion];
     [SDLDebugTool logInfo:logMessage withType:SDLDebugType_RPC toOutput:SDLDebugOutput_All toGroup:self.debugConsoleGroupName];
+    if ([SDLGlobals globals].protocolVersion >= 4) {
```

Comment:
It should remain 4. SDL tracks these requests and v4 apps so it knows what apps it can send LAUNCH_APP and QUERY_APPS system requests: https://github.com/smartdevicelink/sdl_core/blob/master/src/components/application_manager/src/commands/mobile/on_hmi_status_notification_from_mobile.cc#L81-L113